packages/x-date-pickers/src/internals/components/PickerProvider.tsx

```diff
@@ -100,8 +91,24 @@ export interface PickerContextValue<
   TValue extends PickerValidValue,
   TView extends DateOrTimeViewWithMeridiem,
   TError,
-> extends UsePickerValueContextValue<TValue, TError>,
-    UsePickerViewsContextValue<TView> {
+> extends PickerActionsContextValue<TValue, TView, TError> {
+  /**
+   * The current value of the picker.
+   */
+  value: TValue;
+  /**
+   * The timezone to use when rendering the dates.
+   * If a `timezone` prop is provided, it will be used.
+   * If the `value` prop contains a valid date, its timezone will be used.
+   * If no `value` prop is provided, but the `defaultValue` contains a valid date, its timezone will be used.
+   * If no `value` or `defaultValue` is provided, but the `referenceDate` is provided, its timezone will be used.
```

Comment:
could we also detect the users timezone and use that as a backup?

Reply:
We support the `locale` timezone that will be the timezone of the user.
But we set `default` by default, which is the default timezone of the date library, to match the behavior of the date library.